apps/fabric-website/index.html

```diff
@@ -79,39 +105,36 @@
     })(window, document, 'script', 'https://www.google-analytics.com/analytics.js', 'ga');
     ga('create', 'UA-84911004-1', 'auto');
     ga('send', 'pageview');
-</script>
-<script type="text/JavaScript">
-        var varSegmentation = 0;
-        var varClickTracking = 1;
-        var varCustomerTracking = 1;
-        var varAutoFirePV = 1;
-        var Route = "50000";
-        var Ctrl = "SD300";
-        document.write("<script type='text/javascript' src='" + (window.location.protocol) + "//c.microsoft.com/ms.js'" + "'><\/script>");
-    </script>
-<noscript><img alt="" width="1" height="1" src="http://c.microsoft.com/trans_pixel.aspx" /></noscript>
-<script>
+  </script>
+  <script type="text/JavaScript">
+    var varSegmentation = 0; var varClickTracking = 1; var varCustomerTracking = 1; var varAutoFirePV = 1; var Route = "50000";
+    var Ctrl = "SD300"; var script = document.createElement('script'); script.setAttribute('type', 'text/javascript'); script.src
```

Comment:
this is super weird... why getting rid of CRs?

What is ms.js?

what is trans_pixel.aspx?